### PR TITLE
[new release] rtop and reason (3.8.1)

### PR DESCRIPTION
**Pull request by anmonteiro**
Reason toplevel

- Project page: <a href="https://github.com/reasonml/reason">https://github.com/reasonml/reason</a>
- Documentation: <a href="https://reasonml.github.io/">https://reasonml.github.io/</a>

##### CHANGES:

- (Internal) Rename: Reason_migrate_parsetree -> Reason_omp (@ManasJayanth) [reasonml/reason#2666](https://github.com/reasonml/reason/pull/2666)
- Add support for OCaml 5.0 (@EduardoRFS and @anmonteiro) [reasonml/reason#2667](https://github.com/reasonml/reason/pull/2667)

### CODE REVIEW

File: packages/reason/reason.3.8.1/opam [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+homepage: "https://github.com/reasonml/reason"
+doc: "https://reasonml.github.io/"
+bug-reports: "https://github.com/reasonml/reason/issues"
+dev-repo: "git+https://github.com/reasonml/reason.git"
+tags: [ "syntax" ]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml"         {>= "4.03" & < "4.15"}
```
Comment (kit-ty-kate) [inline packages/reason/reason.3.8.1/opam:14]:
```suggestion
  "ocaml"         {>= "4.03" & < "5.1"}
```

File: packages/rtop/rtop.3.8.1/opam [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+homepage: "https://github.com/reasonml/reason"
+doc: "https://reasonml.github.io/"
+bug-reports: "https://github.com/reasonml/reason/issues"
+dev-repo: "git+https://github.com/reasonml/reason.git"
+tags: [ "syntax" ]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml"  {>= "4.02" & < "4.15"}
```
Comment (kit-ty-kate) [inline packages/rtop/rtop.3.8.1/opam:14]:
```suggestion
  "ocaml"  {>= "4.02" & < "5.1"}
```

**Review by kit-ty-kate — COMMENTED** [review]
Your changelog includes "Add support for OCaml 5.0" so I’m guessing that was an oversight

**Comment by anmonteiro** [comment]
Indeed, thanks for fixing! I'll return the fixes upstream.

File: packages/reason/reason.3.8.1/opam [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+doc: "https://reasonml.github.io/"
+bug-reports: "https://github.com/reasonml/reason/issues"
+dev-repo: "git+https://github.com/reasonml/reason.git"
+tags: [ "syntax" ]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml"         {>= "4.03" & < "5.1"}
+  "dune"          {>= "1.4"}
```
Comment (kit-ty-kate) [inline packages/reason/reason.3.8.1/opam:15]:
```suggestion
  "dune"          {>= "2.3"}
```

File: packages/rtop/rtop.3.8.1/opam [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+doc: "https://reasonml.github.io/"
+bug-reports: "https://github.com/reasonml/reason/issues"
+dev-repo: "git+https://github.com/reasonml/reason.git"
+tags: [ "syntax" ]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml"  {>= "4.02" & < "5.1"}
+  "dune"   {>= "1.4"}
```
Comment (kit-ty-kate) [inline packages/rtop/rtop.3.8.1/opam:15]:
```suggestion
  "dune"   {>= "2.3"}
```

**Review by kit-ty-kate — COMMENTED** [review]
Your dune-project file says you require 2.3

**Comment by anmonteiro** [comment]
I clearly messed up the release. Will re-submit.

**Comment by kit-ty-kate** [comment]
Really? It felt like only a couple of missing constraints left to do

**Comment by anmonteiro** [comment]
I didn't intend to bump the release lower bounds. Reason should work on 4.03.

**Comment by kit-ty-kate** [comment]
It worked on 4.03, it just failed with some older versions of its dependencies, the only thing to do is to detect which ones

**Pull request closed without merging**